doc/going-further/helpers/run-php.md

```diff
@@ -18,3 +18,20 @@ the static binary and without conflicts between the external script and internal
 php code of Castor.
 
 The `run_php()` takes exactly the same options as the `run()` function.
+
+### Script requiring more memory
+
+Maybe you need to execute a script that requires more memory than the default
+provided by PHP or castor static binary. In this case, you can add the
+`CASTOR_MEMORY_LIMIT` environment variable to increase the memory limit within
+the context.
```

Comment:
```suggestion
If you need to execute a script that requires more memory than the default
provided by PHP or castor static binary. In this case, add the
`CASTOR_MEMORY_LIMIT` environment variable to increase the memory limit within
the context:
```

Reply:
The sentence is wrong in @lyrixx's patch :smile:

Reply:
Both fixed xD